src/osuTK/Platform/Windows/WinRawJoystick.cs

```diff
@@ -813,7 +816,10 @@ public JoystickState GetState(int index)
                     }
                     else
                     {
-                        return dev.GetState();
+                        lock (dev)
```

Comment:
this is also guaranteed to be an exclusive lock

Reply:
if you're looking to lock on the underlying `FromIndex` instead, you should be doing that inside `Devices`.